prog/dftb+/api/mm/dftbplus.h

```diff
@@ -207,6 +207,16 @@ void dftbp_get_energy(DftbPlus *instance, double *mermin_energy);
 void dftbp_get_gradients(DftbPlus *instance, double *gradients);
 
 
+/**
+ * Queries the stress tensor of the current periodic box
+ *
+ * \param[inout] instance Handler of the DFTB+ instance.
+ *
+ * \param[out] stresstensor Stress Tensor for the periodic box. Shape [3, 3]. Unit: Pascals.
+ */
+void dftbp_get_gradients(DftbPlus *instance, double *stresstensor);
```

Comment:
`dftbp_get_stress_tensor` to match the `bind(C)` from `c_DftbPlus_getStressTensor`

Reply:
That is a good catch, I guess the compiler cannot catch all your copy paste errors after all.

This should be fixed in the current commit.